src/bosh-google-cpi/action/create_vm.go

```diff
@@ -184,7 +185,14 @@ func (cv CreateVM) findZone(zoneName string, disks []DiskCID) (string, error) {
 	return "", fmt.Errorf("Could not find zone %q", zoneName)
 }
 
+func isGcpImageURL(s string) bool {
+	return strings.HasPrefix(s, "https://www.googleapis.com/compute/v1/projects/")
```

Comment:
@dgodd To avoid confusion around sometimes having an Image URL and sometimes having the Image ID, could we also return an ID when ImageURL is used? As long as we use a UUID for the Windows Image Name [as shown here](https://github.com/greenhouse-org/bosh-google-cpi-release/blob/376ae0ce69df64964ac1fff34460f09bdfa3d025/src/bosh-google-cpi/google/image_service/google_image_service_create.go#L41) we should avoid any collisions. Wdyt?